pl-backend/Models/User.cs

```diff
@@ -10,9 +10,9 @@ public class User
         [Required]
         public string? Email { get; set; }
         [Required]
-        public string? PasswordHash { get; set; }
+        public byte[] PasswordHash { get; set; }
         [Required]
-        public string? PasswordSalt { get; set; }
+        public byte[] PasswordSalt { get; set; }
```

Comment:
Those changes require migration, tho there's a problem with reseting schema so we might create our new one. Its something i will take care of later.